base/uk.ac.stfc.isis.ibex.targetplatform/uk.ac.stfc.isis.ibex.targetplatform.target

```diff
@@ -23,7 +23,7 @@
          <repository location="http://download.eclipse.org/releases/2018-09"/>
      </location>
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-		<repository location="http://shadow.nd.rl.ac.uk/ICP_P2/css_gui_dependencies/cs-studio/p2repo/"/>
+		<repository location="http://shadow.nd.rl.ac.uk/ICP_P2/css_gui_dependencies_7212/p2repo/"/>
```

Comment:
Note for reviewer: once you are happy with this PR, please:
- Go to ICP_P2 
- Move `css_gui_dependencies` to `css_gui_dependencies_old`
- Move `css_gui_dependencies_7212` to `css_gui_dependencies`
- Update this line to point at `gui_dependencies` rather than `gui_dependencies_7212`